platforms/core-configuration/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/GradleKotlinDslIntegrationTest.kt

```diff
@@ -1062,6 +1062,29 @@ class GradleKotlinDslIntegrationTest : AbstractKotlinIntegrationTest() {
         )
     }
 
+    @Test
+    fun `can use project layout`() {
+        withProjectRoot(newDir("project")) {
+            withBuildScript(
+                """
+                        println("Settings dir: " + layout.settingsDirectory)
+                        println("Project dir: " + layout.projectDirectory)
+                      """
```

Comment:
💅 Please fix alignment
```suggestion
                """
                println("Settings dir: " + layout.settingsDirectory)
                println("Project dir: " + layout.projectDirectory)
                """
```